desktop/linux/networking.md

```diff
@@ -46,12 +46,15 @@ See [Proxies](index.md#proxies).
 Docker Desktop for Linux allows you to use the host’s SSH agent inside a container. To do this:
 
 1. Bind mount the SSH agent socket by adding the following parameter to your `docker run` command:
-
-    `--mount type=bind,src=/run/host-services/ssh-auth.sock,target=/run/host-services/ssh-auth.sock`
+```console
+$ --mount type=bind,src=/run/host-services/ssh-auth.sock,target=/run/host-services/ssh-auth.sock
+```
```

Comment:
This example is a bit confusing, as it's a "partial" instruction (it's missing the `docker run`); I'm guessing this is why it wasn't put in a `console` block.

The full command would be something like

```console
$ docker run --mount type=bind,src=/run/host-services/ssh-auth.sock,target=/run/host-services/ssh-auth.sock <SOME IMAGE>
```

But perhaps that's too long (and the other options to use depend on what image you're running, etc).

Also looks like the code block needs to be indented to prevent the numbered list from being reset (next item is `1.` again)

Reply:
Had to find a work around so the code block was still scrollable. Currently in the docs the code block crosses over the right-hand toc. I'm sure this is not the proper way to do it 😅